idaes/power_generation/costing/tests/test_NGCC_costing.py

```diff
@@ -0,0 +1,352 @@
+    costing_initialization(m.fs)
+    assert degrees_of_freedom(m) == 0
+
+    # Solve the model
+    results = solver.solve(m, tee=True)
+    assert results.solver.termination_condition == \
+        pyo.TerminationCondition.optimal
+    assert results.solver.status == pyo.SolverStatus.ok
+
+    # Obtain the total plant costs for each account
+    TPC_components = []
+    TPC_components.append(sum(pyo.value(m.fs.b1.costing.total_plant_cost[ac])
+                              for ac in FW_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b2.costing.total_plant_cost[ac])
+                              for ac in RW_withdraw_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b3.costing.total_plant_cost[ac])
+                              for ac in FuelG_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b4.costing.total_plant_cost[ac])
+                              for ac in PW_discharge_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b5.costing.total_plant_cost[ac])
+                              for ac in FG_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b6.costing.total_plant_cost[ac])
+                              for ac in CT_grosspower_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b7.costing.total_plant_cost[ac])
+                              for ac in HRSG_duty_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b8.costing.total_plant_cost[ac])
+                              for ac in Stack_flow_gas_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b9.costing.total_plant_cost[ac])
+                              for ac in Steam_turbine_gross_power_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b10.costing.total_plant_cost[ac])
+                              for ac in Condenser_duty_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b11.costing.total_plant_cost[ac])
+                              for ac in Cooling_tower_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b12.costing.total_plant_cost[ac])
+                              for ac in Circ_water_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b13.costing.total_plant_cost[ac])
+                              for ac in plant_gross_power_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b14.costing.total_plant_cost[ac])
+                              for ac in auxilliary_load_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b15.costing.total_plant_cost[ac])
+                              for ac in stg_ctg_accounts))
+    TPC_components.append(sum(pyo.value(m.fs.b16.costing.total_plant_cost[ac])
+                              for ac in gasturbine_accounts))
+
+    assert pytest.approx(sum(TPC_components), abs=0.5) == 574.85
+
+    #  Testing cost component values for some accounts against the
+    #  NETL baseline report revision 4
+
+    # Accounts with raw water withdrawal as reference parameter
+    assert pytest.approx(sum(pyo.value(m.fs.b2.costing.total_plant_cost[ac])
```

Comment:
It doesn't really matter in the end, but to me it makes more sense to write this as:

```
assert pytest.approx(expected_value, tolerance) == value(variable)
```

It might also help with making the line breaks a bit neater, as the two sides of the `==` are of similar lengths.